### ISSUE TITLE
Feature Request: Ability to dynamically interact with saved windows without having to open them 

### ISSUE THREAD

**Issue by tomasmenezes** [stefanXO/Tab-Manager-Plus#180]
At the moment, saved windows are static in the sense that once saved and closed they need to be subsequently reopened to be editable. Editable saved windows would provide a much better workflow imo, with options such as tab dragging into and out of possibly closed spaces.

**Comment by aamikus** [comment]
Agreed! The first thing I noticed is that I am unable to edit the saved windows. I would like to be able to rename them, maybe drag them around, removed tabs from them etc.

Is there anyway to help with the development of this extension? Can I submit pull requests? @stefanXO

**Comment by octopushugger** [comment]
Came here to request this

**Comment by Osmiridium** [comment]
I agree, this is very much needed for enhanced accessibility and ease-of-use.

**Comment by 0xdevalias** [comment]
Partially related:

- https://github.com/stefanXO/Tab-Manager-Plus/issues/172
